libs/entitlement/asm-provider/src/main/java/org/elasticsearch/entitlement/instrumentation/impl/InstrumenterImpl.java

```diff
@@ -155,10 +158,10 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
                 var key = new MethodKey(className, name, Stream.of(Type.getArgumentTypes(descriptor)).map(Type::getInternalName).toList());
                 var instrumentationMethod = checkMethods.get(key);
                 if (instrumentationMethod != null) {
-                    // System.out.println("Will instrument method " + key);
+                    logger.debug("Will instrument {}", key);
```

Comment:
Can you please split this out? We've talked about it across several PRs, let's do it, but as distinct commits.

Reply:
I thought the controversy was using exceptions to get a stack trace.

Reply:
There's no controversy, I just think adding debug logging to instrumentation is unrelated to adding miscellaneous file entitlements?